payment-circle/src/main/java/org/stellar/anchor/paymentservice/circle/CirclePaymentService.java

```diff
@@ -1,30 +1,41 @@
 package org.stellar.anchor.paymentservice.circle;
 
 import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
 import com.google.gson.JsonObject;
 import io.netty.handler.codec.http.HttpHeaderNames;
 import java.math.BigDecimal;
+import java.util.*;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.UUID;
+import java.util.stream.Collectors;
+import lombok.Getter;
 import org.apache.commons.validator.routines.EmailValidator;
 import org.stellar.anchor.exception.HttpException;
 import org.stellar.anchor.paymentservice.*;
-import org.stellar.anchor.paymentservice.circle.model.CircleBalance;
-import org.stellar.anchor.paymentservice.circle.model.CircleTransactionParty;
-import org.stellar.anchor.paymentservice.circle.model.CircleWallet;
+import org.stellar.anchor.paymentservice.circle.model.*;
 import org.stellar.anchor.paymentservice.circle.model.request.CircleSendTransactionRequest;
 import org.stellar.anchor.paymentservice.circle.model.response.*;
 import org.stellar.anchor.paymentservice.circle.util.NettyHttpClient;
+import org.stellar.sdk.Server;
 import reactor.core.publisher.Mono;
 import reactor.netty.ByteBufMono;
 import reactor.netty.http.client.HttpClient;
 import reactor.netty.http.client.HttpClientResponse;
 import reactor.util.annotation.NonNull;
 import reactor.util.annotation.Nullable;
 
-public class CirclePaymentService implements PaymentService {
-  private static final Gson gson = new Gson();
+public class CirclePaymentService implements PaymentService, StellarReconciliation {
+  private static final Gson gson =
+      new GsonBuilder()
+          .registerTypeAdapter(CircleTransfer.class, new CircleTransfer.Serialization())
+          .registerTypeAdapter(CirclePayout.class, new CirclePayout.Deserializer())
```

Comment:
Whats the point of registering these type adapters if you still have to pass the class to `fromJson()`?